includes/class-newspack-popups.php

```diff
@@ -221,14 +221,35 @@ public static function register_meta() {
 			]
 		);
 
+		// Meta field for all post types.
+		\register_meta(
+			'post',
+			'newspack_popups_has_disabled_popups',
+			[
+				'show_in_rest'  => true,
+				'type'          => 'boolean',
+				'single'        => true,
+				'auth_callback' => '__return_true',
+			]
+		);
 	}
 
 	/**
 	 * Load up common JS/CSS for wizards.
 	 */
 	public static function enqueue_block_editor_assets() {
 		$screen = get_current_screen();
+
 		if ( self::NEWSPACK_PLUGINS_CPT !== $screen->post_type ) {
```

Comment:
Since the text indicates this option is on posts and pages only, do you think we should explicitly enqueue only on those two post types?

Reply:
Done in c18d70f609a3456a508fc90570cb8ec4c7ef8add